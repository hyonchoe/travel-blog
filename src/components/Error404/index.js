import React from 'react'
import './style.css'

const Error404 = () => {
    return (
        <p className='error'>Page not found</p>
    )
}

export default Error404