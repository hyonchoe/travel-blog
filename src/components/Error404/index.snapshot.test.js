import React from 'react'
import { shallow } from 'enzyme'
import Error404 from './'

describe('Error404', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(
            <Error404 />
        )
        expect(wrapper).toMatchSnapshot()
    })
})