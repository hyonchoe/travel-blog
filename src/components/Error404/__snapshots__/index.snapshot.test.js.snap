// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 matches snapshot 1`] = `
<p
  className="error"
>
  Page not found
</p>
`;
